assignments/lambda-classes.js

```diff
@@ -1 +1,222 @@
 // CODE here for your Lambda Classes
+//==== Person
+class Person {
+  constructor(attributes) {
+    this.name = attributes.name
+    this.age = attributes.age
+    this.location = attributes.location
+    this.gender = attributes.gender
+  }
+  speak() {
+    console.log(`Hello my name is ${this.name}, I am from ${this.location}.`)
```

Comment:
These methods should probably be returning a string, rather than console logging. That's why you were getting all those undefined messages, because you were trying to console log a console log. 

```suggestion
    return `Hello my name is ${this.name}, I am from ${this.location}.`
```

Reply:
Yes, I was wondering about that exact same thing. The instructions said to _log out_ the messages but I was thinking that returning a string should be the proper method. 

I'll update the methods and have it return the strings.

Thanks Joe!

Reply:
All suggested changes have been applied. Let me know if any changes are needed. In the meantime I resolve this note. Thanks.